oam/workloads/application.meshery.layer5.io.schema.json

```diff
@@ -58,7 +64,9 @@
                   "default": "http"
                 },
                 "containerPort": {
-                  "type": "integer"
+                  "type": "number",
+                  "minimum": 0,
+                  "maximum": 100
```

Comment:
cantainer port will always be greater than 1024 considering 0-1024 the system resewrved ports. Infact. You should not impose any restriction on container port.

Reply:
Gotcha :+1: